test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature

```diff
@@ -41,3 +41,15 @@ Feature: Status Desktop community
         | community_channel_name    | community_channel_description   |
         | test-channel    | Community channel description tested 1      |
 
+    Scenario Outline: Admin edits a community channel
+    	When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
+		Then the user lands on the community named myCommunity
+		When the admin creates a community channel named test-channel, with description My description
+        Then the user lands on the community channel named test-channel
+        When the admin edits a community channel named <community_channel_name> to the name <new_community_channel_name>
+        # FIXME there is a bug in desktop where the channel switches back to general once we update
```

Comment:
But can we build the test? I mean, if we can correctly build the test we can have this operative and it is our "first" test failure that needs to be addressed in desktop, but the test itself is correct, it fails bc the app doesn't work as expected! WDYT?

Reply:
Ah good point. Yeah the test works, it just fails because desktop had an issue. 
But I fixed the issue in desktop, I'm rebasing right now so it should all work 😄